README.md

```diff
@@ -163,6 +163,119 @@ The script `load.py` (requires Python 3) can be used to restore from such JSON d
 
 The optional parameter `--no-commits` can be added to skip those entries that
 look like a commit ID. Note that this might break previous permalinks.
+
+# CONNECTING JAVA DEBUGGER WITH DOCKER COMPOSE
+
+Connecting to a java debugger can be broken down into the following steps:
+###1. Run the application using docker-compose
+Create a Dockerfile_debug at the root of the directory with the same content as the Dockerfile of the application
+
+Create a docker-compose-debug.yml file with the same content as the docker-compose.yml file at the root of the directory replacing this in the docker-compose-debug.yml
+```yaml
+    image: commonworkflowlanguage/cwlviewer:v1.3.0
+```
+
+with this:
+
+```yaml
+build:
+  context: .
+  dockerfile: Dockerfile_debug
+```
+This is because we want docker to build with our source code on our local instead of the image on docker hub.
+
+In the port section we want to specify a different port to listen to debug connections.
+
+Here, we are using address as 5005.
+
+In the command section as well, we configure JVM to allow debugging and We enable the agent Java Debug Wire Protocol (JDWP) inside the JVM.
+
+At the end you should have something looking like this
+
+```yaml
+build:
+ context: .
+ dockerfile: Dockerfile_debug
+ports:
+ - "8080:8080"
+ - "5005:5005"
+command: java -Djava.security.egd=file:/dev/./urandom -jar /usr/lib/cwlviewer.jar
+```
+If on intellij, open the docker-compose-debug.yml and click on the ![img_2.png](img_2.png)  in the gutter.
```

Comment:
Could you put this under a heading like `### Using IntelliJ` so it is clear these instructions are specific to that IDE.

Reply:
Okay I will. Thanks

Reply:
Hi @tetron, I've added the changes